.github/workflows/ci.yml

```diff
@@ -0,0 +1,80 @@
+on: push
+
+jobs:
+  code-quality:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: erlef/setup-beam@v1
+        with:
+          # Just some default versions to run code quality tools with.
+          # These can be upgraded as and when but probably won't need to change much.
+          otp-version: "24.0.6"
+          elixir-version: "1.12.3"
+      - uses: actions/cache@v2
+        with:
+          path: |
+            deps
+            _build
+            priv/plts
+          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
```

Comment:
It may be worth including the Elixir and OTP versions in the key as well. It may not be super important for code quality, seeing as you will probably only care about the most recent version of elixir.